Clarify the value range of SETTINGS_ENABLE_WEBTRANSPORT (#52)
ietf-wg-webtrans/draft-ietf-webtrans-http3: draft-ietf-webtrans-http3.md

In order to indicate support for WebTransport, both the client and the server
MUST send a SETTINGS_ENABLE_WEBTRANSPORT value set to "1" in their SETTINGS
frame.  The SETTINGS_ENABLE_WEBTRANSPORT parameter value SHALL be either "0" or

> "1", with "0" being the default; an endpoint that receives a value different
> from "0" or "1" MUST close the connection with the H3_SETTINGS_ERROR error

Reviewer: ```suggestion
"1", with "0" being the default; an endpoint that receives a value other
than "0" or "1" MUST close the connection with the H3_SETTINGS_ERROR error
```
